ONSdigital/dp-frontend-filter-dataset-controller
# Search, Renderer Hierarchy clients replaced

### What

Replaced the search, renderer and hierarchy clients
Note this extends the PR https://github.com/ONSdigital/dp-frontend-filter-dataset-controller/pull/165

### How to review

* Check out the code
* Run tests
* Run the service

### Who can review

Anyone except me
